apps/explorer/src/ui/stories/TransactionBlockCard.stories.tsx

```diff
@@ -3,37 +3,34 @@
 
 import { type Meta, type StoryObj } from '@storybook/react';
 
-import {
-	TransactionBlockCard,
-	type TransactionBlockCardProps,
-	TransactionBlockCardSection,
-} from '~/ui/TransactionBlockCard';
+import { CollapsibleCard, type CollapsibleCardProps } from '~/ui/collapsible/CollapsibleCard';
```

Comment:
This file should be renamed to match the name of the story.

Reply:
Updated file name